pnpm-workspace.yaml

```diff
@@ -10,3 +10,18 @@ catalog:
   'source-map-js': ^1.2.1
   'vite': ^5.4.15
   '@vitejs/plugin-vue': ^5.2.3
+
+onlyBuiltDependencies:
+  - '@swc/core'
+  - 'esbuild'
+  - 'puppeteer'
+  - 'simple-git-hooks'
+  - 'unrs-resolver'
```

Comment:
![image](https://github.com/user-attachments/assets/a62e013c-7e6e-461d-ae31-2b4e1a514608)

`unrs-resolver` is a dependency of `eslint-plugin-import-x`, and the `postinstall` script was added in version 1.6.0.